KTLOSNaxxLoot.json

```diff
@@ -613,17 +613,17 @@
 	},
 	"Kel'Thuzad":{
 		"The Phylactery of Kel'Thuzad":{
-			"prio":["Killstep(sunnyD)","ThePrestige(crush)","Johnscar(sunnyD)","LC","EP/GP"],
+			"prio":["Johnscar(sunnyD)","Killstep(sunnyD)","LC","EP/GP"],
 			"gp":"6",
 			"wowID":"22520"
 		},
 		"The Hungering Cold":{
-			"prio":["Frau(2)","R1 Loot Council","Snydicate(1)","Dill(1)","Vajeen(1)","LC","EP/GP"],
+			"prio":["Frau(2)","Bendriller(flex)","Snydicate(1)","Dill(1)","Vajeen(1)","LC","EP/GP"],
 			"gp":"18",
```

Comment:
Same comment, I'm ok with (2) but you shouldn't get the next THC in r1

Reply:
Personally I'm not planning to swap back raids any time soon, but I am a little unsettled by the idea of Brienne getting thc early after joining so late or Dill and Vajeen having gressil/thc before I get a single KT weapon. I haven't taken a weapon from this guild since Perds and while I'm not adamant on getting a KT weapon early, moving me behind Dill for thc from raid 1 is something Dill and I would have to discuss.

Reply:
Then (2) makes the most sense if you're not swapping, and when you do swap, there needs to be another conversation to be had.  You're either 2nd or 3rd on Gressil, and potentially 2nd on THC which is very good prio for someone who's current median parse is 56.

Reply:
I talked with Dill and given the way he framed it, I think committing to sunnyD full clearing is the right thing to do. If sunnyD isn't full clearing by the time Snydicate gets a thc I may want to revisit this issue. It would be nice if other class leads would bring their caster dps (mage specifically) to raid 2 to shore up that weakness and help us kill KT.